runtime/bindings/python/tests/test_ngraph/test_preprocess.py

```diff
@@ -0,0 +1,293 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+import openvino as ov
+import openvino.opset8 as ops
+from openvino.impl.preprocess import PrePostProcessor, InputInfo, PreProcessSteps, InputTensorInfo, \
+    OutputTensorInfo, OutputNetworkInfo, InputNetworkInfo, ColorFormat, OutputInfo, \
+    PostProcessSteps, ResizeAlgorithm
+from openvino.impl import Function, Output, Type
+from openvino.utils.decorators import custom_preprocess_function
+from tests.runtime import get_runtime
+
+
+def test_ngraph_preprocess_mean():
+    shape = [2, 2]
```

Comment:
Can you please add a test for several model with several inputs?

Reply:
Added.